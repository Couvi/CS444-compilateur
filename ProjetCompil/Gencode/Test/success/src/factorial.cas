proGram
	Result : integer;
	Counter : integer;
	N : integer;
	I : 1 .. 13;
BegiN
	N := 7;
	Result := N;
	Counter := N - 1;
	for I := Counter downto 1 do
		Result := Result * I;
		write(Result, " ", I);
		new_line;
	end;
	write(Result);
end.
