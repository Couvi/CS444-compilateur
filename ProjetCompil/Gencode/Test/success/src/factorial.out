42 6
210 5
840 4
2520 3
5040 2
5040 1
5040
